src/styles/main.scss

```diff
@@ -0,0 +1,86 @@
+$day-size: 100px;
+$border-color: #000;
+$backgroundColor: #eee;
+$backgroundColorHover: #ffbfcb;
```

Comment:
Variable names should be written in kebab-case. Please change `$backgroundColorHover` to `$background-color-hover` to adhere to the code style rules.